packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react';
+import classed from '../../lib/classed';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import AuthOptions, { AuthDisplay } from './AuthOptions';
+import { AuthTriggers } from '../../lib/auth';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const Container = classed('div', 'flex flex-col w-[23.25rem]');
+
+export function AuthenticationBanner(): ReactElement {
+  const { showLogin } = useAuthContext();
+
+  return (
+    <div
+      className="fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center gap-6 py-10"
+      style={{
```

Comment:
Custom colors?
Maybe double verify with Tsahi, probably a design mistake?

Reply:
Sent a Slack message. Will wait for their response 🚢

Reply:
Found some equivalent values in our codebase, will update quickly.

Reply:
Applied the missing styles 🥳 

![Screenshot 2024-02-15 at 5 59 26 PM](https://github.com/dailydotdev/apps/assets/13744167/f194d6df-119d-4f1d-b478-ad2cfcfbe1a6)

Reply:
Not urgent, but would it be possible to use the tailwind variation of this?
If too much work never mind :)

Reply:
Let's see, I will check tailwinds docs for linear gradient as it can definitely be used again in the future.

Reply:
Doesn't look exactly the same after converting. Not sure if we go for this.

![Screenshot 2024-02-15 at 6 35 35 PM](https://github.com/dailydotdev/apps/assets/13744167/6cf6027e-b4b6-41a7-90dd-7520e71bf44e)

Reply:
Merging now to unblock, but so far this is what I have tried:

```
 <BottomBannerContainer
      // style={style}
      className="gap-6 bg-gradient-to-l from-theme-overlay-active-cabbage from-0% via-theme-overlay-active-onion via-100% bg-theme-bg-primary py-10 shadow-3"
    >
```

I've also tried adding `to-theme-bg-primary` but doesn't change anything since the stop mentions 0 to 100 already. Not sure how it behaves differently on CSS style.

Reply:
Found the issue and fixed here: https://github.com/dailydotdev/apps/pull/2638